lib/logger.js

```diff
@@ -50,57 +52,53 @@ function trimTo(str, amt) {
   return result;
 }
 
-function Logger(cons) {
-  var logger = cons || console;
-  log = function () {
-    logger.log.apply(logger, arguments);
+function Logger(logger = console) {
+  this.log = (...args) => {
+    logger.log.bind(logger, ...args);
   };
 
-  log.cols = function () {
-    var width = process.stdout.columns;
-    var pads = 0;
-    var padsWidth = 0;
-    var cols = 0;
-    var colsWidth = 0;
-    var input = arguments;
-
-    for (var h = 0; h < arguments.length; ++h) {
-      if (typeof arguments[h] === 'number') {
-        padsWidth += arguments[h];
-        pads++;
+  this.log.cols = (...args) => {
+    const width = process.stdout.columns;
+    const input = args;
+    let pads = 0;
+    let padsWidth = 0;
+
+    for (let h = 0; h < args.length; h += 1) {
+      if (typeof args[h] === 'number') {
+        padsWidth += args[h];
+        pads += 1;
       }
-      if (_.isArray(arguments[h]) && typeof arguments[h][0] === 'number') {
-        padsWidth += arguments[h][0];
-        pads++;
+      if (Array.isArray(args[h]) && typeof args[h][0] === 'number') {
+        padsWidth += args[h][0];
+        pads += 1;
       }
     }
 
-    cols = arguments.length - pads;
-    colsWidth = Math.floor((width - padsWidth) / cols);
-
-    var lines = [];
+    const cols = args.length - pads;
+    const colsWidth = Math.floor((width - padsWidth) / cols);
+    const lines = [];
 
-    var go = function () {
-      var str = '';
-      var done = true;
-      for (var i = 0; i < input.length; ++i) {
+    const go = () => {
```

Comment:
If you can, try and move this function into a method. Maybe something like `render(input)`.